src/app/core/components/breadcrumbs/breadcrumbs.component.spec.ts

```diff
@@ -50,12 +50,12 @@ describe('BreadcrumbsComponent', () => {
               data: { title: 'Path1' },
               routeConfig: { path: 'path1' },
               url: [{ path: 'path1' }],
-              firstChild: { data: { title: 'Path2' }, routeConfig: { path: 'path2' }, url: [{ path: 'path2' }] }
-            }
-          }
-        }
-      }
-    ]
+              firstChild: { data: { title: 'Path2' }, routeConfig: { path: 'path2' }, url: [{ path: 'path2' }] },
+            },
+          },
+        },
+      },
+    ],
   ])('should return %o when router state is %o', async (expected: { label: string; path: string }[], routeState: any) => {
```

Comment:
A way to remove the `any` ?

Reply:
I can't seem to get it to resolve in the editor when trying to replace it, the selector suggests that it should be of type  RouterReducerState<SerializedRouterStateSnapshot> but the above object flags up as not being able to cast to it